Packs/Yara/Scripts/YaraScan/YaraScan.py

```diff
@@ -17,7 +17,7 @@ def main():
 
     fileInfos = []
     for item in entryIDs:
-        res = demisto.executeCommand("getFilePath", {"id": item})
+        res = demisto.executeCommand("getFilePath", {"id": item, "incidentId": item.split("@")[1]})
```

Comment:
```suggestion
        item_params = {"id": item}
        try:
            entry_incident_id = item.split("@")[1]
            item_params["incidentId"] = entry_incident_id
        except:
            pass

        res = demisto.executeCommand("getFilePath", item_params)
```
This will ensure existing logic keeps working even if the parsing fails for some reason.